solana/programs/token_bridge_relayer/src/constants.rs

```diff
@@ -7,3 +7,8 @@ pub const SEED_PREFIX_BRIDGED: &[u8] = b"bridged";
 #[constant]
 /// AKA `b"tmp"`.
 pub const SEED_PREFIX_TMP: &[u8] = b"tmp";
+
+#[constant]
+/// Swap rate precision. This value should NEVER change, unless other Token
+/// Bridge Relayer contracts are deployed with a different precision.
+pub const SWAP_RATE_PRECISION: u32 = 100_000_000;
```

Comment:
should we consider a larger precision and using a wider numeric type to represent values?

Reply:
Unfortunately, I think we're stuck with this precision given that the EVM/Sui contract already deployed with this value. I think it could be potentially dangerous to not synchronize these values across different blockchains.

Reply:
The deployer should consider this in the future when rolling out a new version of these contracts on EVM/Sui/Solana.